*** src/ucs.cpp.org	2008-09-16 00:19:21.000000000 +0900
--- src/ucs.cpp	2009-10-13 18:59:06.000000000 +0900
***************
*** 178,184 ****
          tmp == "shift_jis" || tmp == "cp932")
        return CP932;
      else if (tmp == "euc"   || tmp == "euc_jp" ||
!              tmp == "euc-jp" || tmp == "euc-jp-win" ||
               tmp == "euc-jp-ms" || tmp == "eucjp-ms")
        return EUC_JP;
      else if (tmp == "utf8" || tmp == "utf_8" ||
--- 178,184 ----
          tmp == "shift_jis" || tmp == "cp932")
        return CP932;
      else if (tmp == "euc"   || tmp == "euc_jp" ||
!              tmp == "euc-jp" || tmp == "euc-jp-win" || tmp == "eucjp-win" ||
               tmp == "euc-jp-ms" || tmp == "eucjp-ms")
        return EUC_JP;
      else if (tmp == "utf8" || tmp == "utf_8" ||
